packages/web3-shared/evm/hooks/useGasPrice.ts

```diff
@@ -2,7 +2,7 @@ import { useAsyncRetry } from 'react-use'
 import { useWeb3 } from '.'
 
 export function useGasPrice() {
-    const web3 = useWeb3()
+    const web3 = useWeb3(true)
```

Comment:
Using our default RPC to query gas price since there's some issue with Fortmatic RPC.